pos_event_calendar/models/event_event.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025 Moka
+# @author Damien Horvat <damien@moka.cloud>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+class EventEvent(models.Model):
+    """Custom model to load inside the POS"""
+
+    _name = "event.event"
+    _inherit = ["event.event", "pos.load.mixin"]
+
+    @api.model
+    def _load_pos_data_fields(self, config_id):
+        """Specify which fields to load in the POS"""
+
+        super()._load_pos_data_fields(config_id)
+        return [
+            "id",
+            "name",
+            "date_begin",
+            "date_begin_located", 
+            "date_end",
+            "date_end_located",
+            "seats_limited",
+            "seats_max",
+            "seats_available",
+            "address_id",
+            "organizer_id",
+            "event_type_id",
+            "active",
+            "company_id",
+            "event_ticket_ids",
+            "image_1024",
+            "tag_ids",
+            "question_ids",
+            "general_question_ids",
+            "specific_question_ids",
+            "badge_format",
+        ]
```

Comment:
Do not fully overwrite the response.
Respect whatever is return by super, and add only the extra fields you need on top

```suggestion
        res = super()._load_pos_data_fields(config_id)
        return res + [ "date_begin", ... ]
```

This makes the module play nice with other potential modules also overriding this method